lib/commands/install.ts

```diff
@@ -21,26 +16,7 @@ export class InstallCommand implements ICommand {
 
 	private installProjectDependencies(): IFuture<void> {
 		return (() => {
```

Comment:
you can modify `installProjectDependencies` in the following way, in order to skip creation of additional FiberFuture:
```TypeScript
private installProjectDependencies(): IFuture<void> {
    this.$pluginsService.ensureAllDependenciesAreInstalled();
}
```